server/game/cards/01_SOR/units/RelentlessKonstantinesFolly.ts

```diff
@@ -0,0 +1,42 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { CardType, RelativePlayer, Location } from '../../../core/Constants';
+import { StateWatcherRegistrar } from '../../../core/stateWatcher/StateWatcherRegistrar';
+import { CardsPlayedThisPhaseWatcher } from '../../../stateWatchers/CardsPlayedThisPhaseWatcher';
+
+export default class RelentlessKonstantinesFolly extends NonLeaderUnitCard {
+    private cardsPlayedThisPhaseWatcher: CardsPlayedThisPhaseWatcher;
+
+    protected override getImplementationId() {
+        return {
+            id: '3401690666',
+            internalName: 'relentless#konstantines-folly'
+        };
+    }
+
+    protected override setupStateWatchers(registrar: StateWatcherRegistrar): void {
+        this.cardsPlayedThisPhaseWatcher = AbilityHelper.stateWatchers.cardsPlayedThisPhase(registrar, this);
+    }
+
+    public override setupCardAbilities() {
+        this.addConstantAbility({
+            title: 'The first event played by each opponent each round loses all abilities',
+            ongoingEffect: AbilityHelper.ongoingEffects.blank(false),
+            //An event card goes to the discard before its effects resolve, so filtering for being in the discard is appropriate to allow playing an event but blanking its effects.
+            //Filtering for anywhere(WildcardLocation.Any), causes the event card to lose the ability to even play it in the first place.
```

Comment:
nit: can you add a space after the `//` for these lines for readability. again, the linter should be catching this, I need to investigate